forms-flow-web/src/helper/regExp/inputValidator.js

```diff
@@ -0,0 +1,25 @@
+const inputValidator = (inputText) => {
+    // Set a flag to indicate whether the input is valid or not
+    let valid = true;
+    // Iterate over each character in the inputText string using a for...of loop
+    for (const char of inputText) {
```

Comment:
InputText is empty or null check is missing right.? Or is it handled before calling the function

Reply:
it is handled on befor the function call.